Lombiq.Hosting.Tenants.Admin.Login/Views/TenantAdminShape.cshtml

```diff
@@ -9,7 +9,7 @@
 
     var hostString = host != null ? new HostString(host) : Context.Request.Host;
 
-    var actionPath = $"/Lombiq.Hosting.Tenants.Admin.Login/TenantLogin/Index";
+    const string actionPath = "/Lombiq.Hosting.Tenants.Admin.Login/TenantLogin/Index";
```

Comment:
Why are you using `const` here? And if we using `const` I think it should be above other variables.

Reply:
It **is** a constant string, but I've reverted it to `var` for the sake of simplicity.